Makefile

```diff
@@ -31,7 +31,7 @@ examples:
 		@echo "OK 👌"
 
 benchmark:
-		@cd instrumented-tests/Benchmark && $(MAKE)
+		@xcodebuild -workspace "Datadog.xcworkspace" -scheme "DatadogIntegrationTests" test -destination "name=iPhone 11 Pro Max"
```

Comment:
Do we still need `make benchmark`? This tests are run through `bitrise.yml` in much nicier way with `xcpretty` log. Also we duplicate the `iPhone 11 Pro Max` instrumentation here, I think it's better to centralize it with `bitrise.yml` only. Now there's IMO no need to do anything for integration tests in `Makefile`.